### PR TITLE
Feature/gfsv16b

### PR DESCRIPTION
**Pull request by lgannoaa**
ecflow GFSv16 workflow NCO code delivery package merge.
This version of ecflow is configured with resource requirement based on HWM value for running GFSv16 in NCO production.

### CODE REVIEW

**Review by KateFriedman-NOAA — APPROVED** [review]
Changes look good and were tested by Lin. Doesn't impact rocoto parallel.

File: ecflow/ecf/scripts/gdas/atmos/post/jgdas_atmos_post_anl.ecf [hunk 1]
```diff
@@ -3,8 +3,8 @@
 #BSUB -P %PROJ%
 #BSUB -J %E%gdas_atmos_post_%FHR%_%CYC%
 #BSUB -o %COM%/output/%ENVIR%/today/gdas_atmos_post_%FHR%_%CYC%.o%J
-#BSUB -W 0:10
-#BSUB -n 56
+#BSUB -W 2:00
```
Comment (aerorahul) [inline ecflow/ecf/scripts/gdas/atmos/post/jgdas_atmos_post_anl.ecf:6]: 2 hours, Really?
Are we sleeping in this script and waiting for a file to fall in place?

File: ecflow/ecf/scripts/gdas/atmos/post/jgdas_atmos_post_f009.ecf [hunk 2]
```diff
@@ -3,8 +3,8 @@
 #BSUB -P %PROJ%
 #BSUB -J %E%gdas_atmos_post_%FHR%_%CYC%
 #BSUB -o %COM%/output/%ENVIR%/today/gdas_atmos_post_%FHR%_%CYC%.o%J
-#BSUB -W 0:10
-#BSUB -n 56
+#BSUB -W 2:00
```
Comment (aerorahul) [inline ecflow/ecf/scripts/gdas/atmos/post/jgdas_atmos_post_f009.ecf:6]: 2 hours is an awful long time to reserve a job for a single forecast hour to be postprocessed.
The scheduler should not submit this job until the file is in place for it to work on it.

File: ecflow/ecf/scripts/gfs/atmos/post/jgfs_atmos_post_f001.ecf [hunk 3]
```diff
@@ -4,7 +4,7 @@
 #BSUB -J %E%gfs_atmos_post_%FHR%_%CYC%
 #BSUB -o %COM%/output/%ENVIR%/today/gfs_atmos_post_%FHR%_%CYC%.o%J
 #BSUB -W 6:00
```
Comment (aerorahul) [inline ecflow/ecf/scripts/gfs/atmos/post/jgfs_atmos_post_f001.ecf:6]: 6 hours for a single forecast file to process is too long?
Does the task in here take 6 hours?

There are 384 hours of forecasts to process.
Do we need 384 `jgfs_atmos_post_fHHH.ecf` files to maintain that?

**Review by aerorahul — CHANGES_REQUESTED** [review]
There is so much abuse of resources in this PR.
2 hours for a jgdas single forecast hour to process is too long.
6 hours for a jgfs single forecast hour to process is way too long.

There are 9 GDAS forecast hours and 384 GFS forecast hours to process and a separate `.ecf` file for each of it. 
This should be reduced to 1 file for GDAS and 1 for GFS.

**Comment by yangfanglin** [comment]
NCO lunches each post job as an independent job.  Yes, a single script can (should) work for all forecast hours.  But this has not been pursued so we will stick with what we have.
For CUP hours assign to the post jobs, Lin, please reduce each to 20 minutes.   Those 6 hours and 2 hours are used by EMC parallels in which post is running differently.

**Pull request merged**